ingestion-edge/tests/unit/publish/test_init_app.py

```diff
@@ -4,66 +4,12 @@
 from sanic.request import Request
 from unittest import mock
 import pytest
-
-ROUTE_TABLE = [
-    Route("/stub/<suffix:path>", "stub_installer", ("GET",)),
-    Route("/submit/telemetry/<suffix:path>", "telemetry"),
-    Route("/submit/sslreports", "tls_error_reports", ("post", "put")),
-    Route("/submit/<suffix:path>", "ingestion", ("PoSt", "pUt")),
-    Route("/submit", "ingestion"),
-]
-
-
-@pytest.fixture
-def app():
-    app = Sanic(name="test", register=False)
-    app.config.update(QUEUE_PATH=":memory:", METADATA_HEADERS={}, ROUTE_TABLE=[])
-    return app
-
-
-@pytest.mark.parametrize(
-    "uri_bytes,method,kwargs",
-    [
-        (
-            b"/stub/some/path/with=stuff",
-            "GET",
-            {"topic": "stub_installer", "suffix": "some/path/with=stuff"},
-        ),
-        (
-            b"/submit/telemetry/path/with=stuff",
-            "POST",
-            {"topic": "telemetry", "suffix": "path/with=stuff"},
-        ),
-        (
-            b"/submit/some/path/with=stuff",
-            "PUT",
-            {"topic": "ingestion", "suffix": "some/path/with=stuff"},
-        ),
-        (b"/submit/sslreports", "POST", {"topic": "tls_error_reports"}),
-        (b"/submit", "PUT", {"topic": "ingestion"}),
-    ],
-)
-async def test_endpoint(app, kwargs, method, mocker, uri_bytes):
```

Comment:
i removed this test because it was reaching into sanic internals that no longer exist.